components/pkg-export-kubernetes/src/manifest.rs

```diff
@@ -46,7 +46,7 @@ impl Manifest {
         let count = matches.value_of("COUNT").unwrap_or("1").parse()?;
         let topology: Topology = FromStr::from_str(
             matches.value_of("TOPOLOGY").unwrap_or("standalone"),
-        ).unwrap_or(Topology::Standalone);
+        ).unwrap_or(Default::default());
```

Comment:
You probably could have `let topology = FromStr(…).unwrap_or(Topology::default())`, no? (Without the `: Topology` part after the variable name).

Reply:
I think something along the lines of `Topology::from_str` should work.